File: cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/validation/FileSystemValidationResult.java
package com.sequenceiq.cloudbreak.cloud.event.validation;

import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;

public class FileSystemValidationResult extends CloudPlatformResult {

    public FileSystemValidationResult(Long resourceId) {
        super(resourceId);
    }

    public FileSystemValidationResult(String statusReason, Exception errorDetails, Long resourceId) {
        super(statusReason, errorDetails, resourceId);
    }
}

File: cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/validation/FileSystemValidationResult.kt
package com.sequenceiq.cloudbreak.cloud.event.validation

import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformRequest
import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult

class FileSystemValidationResult : CloudPlatformResult<CloudPlatformRequest<Any>> {

    constructor(request: CloudPlatformRequest<Any>) : super(request) {
    }

    constructor(statusReason: String, errorDetails: Exception, request: CloudPlatformRequest<Any>) : super(statusReason, errorDetails, request) {
    }
}
